scootapi/server/server_test.go

```diff
@@ -0,0 +1,86 @@
+package server
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/scootdev/scoot/common/stats"
+	"github.com/scootdev/scoot/saga/sagalogs"
+	"github.com/scootdev/scoot/sched"
+	"github.com/scootdev/scoot/sched/scheduler"
+	"github.com/scootdev/scoot/scootapi/gen-go/scoot"
+)
+
+// ensure a scheduler initializes to the correct state
+func Test_RequestCounters(t *testing.T) {
+
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	s := scheduler.NewMockScheduler(mockCtrl)
+	s.EXPECT().ScheduleJob(gomock.Any()).Return("mockJobId", nil)
+	s.EXPECT().KillJob(gomock.Any()).Return(nil)
+	sc := sagalogs.MakeInMemorySagaCoordinator()
+	statsRegistry := stats.NewFinagleStatsRegistry()
+
+	statsReceiver, _ := stats.NewCustomStatsReceiver(func() stats.StatsRegistry { return statsRegistry }, 0)
+
+	handler := NewHandler(s, sc, statsReceiver)
+
+	domainJobDef := sched.GenJobDef(1)
+	if len(domainJobDef.Tasks[0].Argv) == 0 {
+		domainJobDef.Tasks[0].Argv = []string{"sampleArg"}
+	}
+
+	scootJobDef, _ := schedJobDefToScootJobDef(&domainJobDef)
+
+	_, err := handler.RunJob(scootJobDef)
+	if err != nil {
+		t.Errorf("RunJob returned err:%s", err.Error())
+	}
+
+	_, err = handler.GetStatus("testJobId")
+	if err != nil {
+		t.Errorf("GetStatus returned err:%s", err.Error())
+	}
+
+	_, err = handler.KillJob("testJobId")
+	if err != nil {
+		t.Errorf("GetStatus returned err:%s", err.Error())
+	}
+
+	stats.VerifyStats("", statsRegistry, t,
+		map[string]stats.Rule{
+			"runJobRpmCounter":        {Checker: stats.Int64EqTest, Value: 1},
+			"runJobLatency_ms.avg":    {Checker: stats.FloatGTTest, Value: 0.0},
```

Comment:
not sure what you mean.... this is testing that the latency value was created and it got some value.  We can't be more precise on the value because of variable runtimes.  (we could ballpark it ... > 1000, but then the test may start failing if it ends up on a super fast machine or we modify the logic so the action runs faster).  
Note: this test is using apriori knowledge that the test stat structure create a bunch of field (avg, p50, p...) for each named latency measurement.  I'm using the avg field to validate that a value got into latency - didn't see the need to validate the histogram implementation within latency.